packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/LinkedRdfSourcesAsyncRdfIterator.ts

```diff
@@ -176,6 +176,7 @@ export abstract class LinkedRdfSourcesAsyncRdfIterator extends BufferedIterator<
     let receivedMetadata = false;
 
     // Attach readers to the newly created iterator
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
```

Comment:
This came up as an eslint error after generating a new lockfile

Reply:
👌